tests/components/media_player/test_device_trigger.py

```diff
@@ -67,6 +72,32 @@ async def test_get_triggers(hass, device_reg, entity_reg):
     assert_lists_same(triggers, expected_triggers)
 
 
+async def test_get_trigger_capabilities(hass, device_reg, entity_reg):
+    """Test we get the expected capabilities from an alarm_control_panel."""
+    config_entry = MockConfigEntry(domain="test", data={})
+    config_entry.add_to_hass(hass)
+    device_entry = device_reg.async_get_or_create(
+        config_entry_id=config_entry.entry_id,
+        connections={(device_registry.CONNECTION_NETWORK_MAC, "12:34:56:AB:CD:EF")},
+    )
+    entity_reg.async_get_or_create(DOMAIN, "test", "5678", device_id=device_entry.id)
+    hass.states.async_set(
+        "alarm_control_panel.test_5678", "attributes", {"supported_features": 15}
```

Comment:
```suggestion
        "media_player.test_5678", "attributes", {"supported_features": 15}
```
Do we need to set supported features?

Reply:
No, it's a copy-paste mistake.